apps/ledger-live-desktop/tests/page/ledgerSync.page.ts

```diff
@@ -0,0 +1,102 @@
+import { AppPage } from "tests/page/abstractClasses";
+import { step } from "tests/misc/reporters/step";
+import { expect } from "@playwright/test";
+
+export class LedgerSyncPage extends AppPage {
+  private syncAccountsButton = this.page.getByRole("button", { name: "Sync your accounts" });
+  private closeLedgerSyncButton = this.page.getByRole("button", { name: "Close" });
+  private manageInstancesButton = this.page
+    .getByTestId("walletSync-manage-instances")
+    .getByText("Manage");
+  private manageBackupButton = this.page.getByTestId("walletSync-manage-backup");
+  private deleteBackupButton = this.page.getByTestId("walletSync-manage-backup-delete");
+  private confirmBackupDeletionButton = this.page.getByRole("button", { name: "Delete" });
+  private successTextElement = this.page.locator("span", { hasText: "Success" }).first();
+  private closeLSDrawerButton = this.page.getByTestId("drawer-close-button");
+  private backupDeletionSuccessText = this.page.getByText(
+    "Your devices have been unsynchronized and your key has been deleted",
+  );
+
+  async SyncAccounts() {
+    await this.syncAccountsButton.click();
+  }
+
+  async doesSyncAccountsButtonExist(): Promise<boolean> {
+    return this.syncAccountsButton.isVisible();
+  }
+
+  async closeLedgerSync() {
+    await this.closeLedgerSyncButton.click();
+  }
+
+  async openManageInstances() {
+    await this.manageInstancesButton.click();
+  }
+
+  async manageBackup() {
+    await this.manageBackupButton.click();
+  }
+
+  async deleteBackup() {
+    await this.deleteBackupButton.click();
+  }
+
+  async confirmBackupDeletion() {
+    await this.confirmBackupDeletionButton.click();
+  }
+
+  async destroyTrustchain() {
+    await this.manageBackup();
+    await this.deleteBackup();
+    await this.confirmBackupDeletion();
+  }
+
+  async checkSynchronizationSuccess(): Promise<boolean> {
+    await this.successTextElement.waitFor({ state: "visible" });
+    return this.successTextElement.isVisible();
+  }
+
+  async backupDeletionSuccess(): Promise<boolean> {
+    await this.backupDeletionSuccessText.waitFor({ state: "visible" });
+    return this.backupDeletionSuccessText.isVisible();
+  }
+
+  async countSyncedInstances(): Promise<number> {
+    const countInstancesText = await this.page.getByText("Synchronized instance").textContent();
+    const instanceCount = extractNumberFromText(countInstancesText || "");
+    return instanceCount;
+  }
+
+  async closeLSManagerDrawer() {
+    await this.closeLSDrawerButton.nth(0).click();
+  }
+
+  @step("Check if sync entry point exists")
+  async expectSyncAccountsButtonExist() {
```

Comment:
this looks like overnegineering to me (expectSyncAccountsButtonExist + doesSyncAccountsButtonExist), please keep only one method that check button visibility. Check other page classes for example